addon/components/bs-navbar/toggle.js

```diff
@@ -0,0 +1,19 @@
+import BsButtonComponent from 'ember-bootstrap/components/bs-button';
+import layout from '../../templates/components/bs-navbar/toggle';
+
+/**
+ * Component to implement the responsive menu toggle behavior in a [Components.Navbar](Components.Navbar.html)
+ * component. Have a look there for examples.
+ *
+ * @class NavbarToggle
+ * @namespace Components
+ * @extends Components.Button
+ * @public
+ */
+export default BsButtonComponent.extend({
+  layout,
+
+  classNames: ['navbar-toggle', 'collapsed'],
+  classNameBindings: ['collapsed::active'],
```

Comment:
How come you are using the active class here? I have not found this to be documented on the bootstrap docs.